### PR TITLE
[feat] Data persistence

### PR DESCRIPTION
**Pull request by Hamdrive**
## Overview

Successfully implemented data persistence for the tasks of user. Now user can refresh, leave page and return and tasks will still remain using localStorage.
The title of the page on the pomodoro counter also persists, and dynamically changes as the timer counts and mode changes

Features:
1. User will always have their tasks persist refresh or closing/reloading page
2. Addition of return button to pomodoro timer to return to tasks
3.  Title of pomodoro page dynamically updates with time of counter and the mode

## Deployment

https://drivers-pomodoro-2jao8if0n-hamdrive.vercel.app/

### CODE REVIEW

File: src/components/Navbar/Navbar.jsx [hunk 1]
```diff
@@ -31,7 +31,7 @@ export function Navbar() {
                 <div className="fab fa-github px-sm"></div>Github
               </a>
             </li>
-            {true ? (
+            {/* {true ? (
```
Comment (shivampandey0) [inline src/components/Navbar/Navbar.jsx:34]: Remove unused code comments

**Review by shivampandey0 — COMMENTED** [review]
Clean code with individual components.
Good to merge

**Review by abhij1607 — COMMENTED** [review]
Great work hamza | | Overall clean code and UI

Apart from Shivam's suggestion, everything looks great

File: src/components/Navbar/Navbar.jsx (hunk 1 above)
Comment (Hamdrive) [reply src/components/Navbar/Navbar.jsx:34]: It has been removed in the other PR

**Comment by Hamdrive** [comment]
Successfully reviewed by 2 team mates, no conflicts found.
Merging to dev branch and closing PR

**Pull request merged**